docker/Dockerfile.release

```diff
@@ -8,7 +8,7 @@ RUN npm install -g @subql/cli && npm ci
 COPY . ./
 RUN npm run codegen && npm run build
 
-FROM availj/subquery-node-avail:v4.3.0 as runner
+FROM onfinality/subql-node:v5.2.3 as runner
```

Comment:
There was a reason to build `availj/subquery-node-avail` image, but TBH I forgot.
Just wonder if this could lead to any issues?
I'll check how did we generate this img and reproduce it to `v5.2.3`.
I remember that `availj/subquery-node-avail` is based on `onfinality/subql-node` but with a couple of extra deps...